src/resources/vue/pages/index.vue

```diff
@@ -1,8 +1,16 @@
 <template>
```

Comment:
Dont put the modal in the index page, its not needed there

Reply:
fixed